.github/workflows/dd-build.yaml

```diff
@@ -8,18 +8,67 @@ on:
     - "*-dd*"
 permissions: write-all
 jobs:
-  build:
-    runs-on: ubuntu-latest
+  build-arm64:
```

Comment:
Nit we should define `build-amd64` before `build-arm64` to respect the alphabetical order and for consistency with `needs: [build-amd64, build-arm64]`